js/game-page.js

```diff
@@ -257,11 +259,31 @@ H5P.ArithmeticQuiz.GamePage = (function ($, UI, ArithmeticType, QuestionsGenerat
     this.$button = UI.createButton({
       text: number,
       click: function () {
+        self.announce();
         self.trigger('answered');
+        setTimeout(self.dropLive, 500);
       }
     });
 
+    this.dropLive = function() {
+      var node = self.$liveRegion[0];
+      node.parentNode.removeChild(node);
+    };
+
+    this.announce = function() {
+      self.$liveRegion = $('<div>', {
+        'class': 'h5p-baq-live-feedback',
+        'aria-live': 'assertive',
+        'width': '1px',
+        'height': '1px',
+        text: self.correct ? 'correct' : 'incorrect'
```

Comment:
strings should be translatable, this can be done through placing them in semantics.json